env.example.yml

```diff
@@ -1,9 +1,8 @@
 LAMBDA_EXEC_SG: Insert AWS Security Group ID Here (it must be in the same VPC as the subnet)
 LAMBDA_EXEC_SUBNET: Insert AWS Subnet ID Here (it must be in the same VPC as the security group)
 BUGSNAG_NOTIFIER_KEY: INSERT BUGSNAG NOTIFIER KEY HERE
-HERE_API_KEY: INSERT HERE API KEY HERE
-GEOCODE_EARTH_API_KEY: INSERT GEOCODE.EARTH API KEY HERE
-GEOCODE_EARTH_URL: https://api.geocode.earth/v1
+GEOCODER_API_KEY: INSERT HERE API KEY HERE
```

Comment:
Don't forget to make a breaking change commit (if you use `mastarm commit` to do it it'll get picked up by the semantic release)

Reply:
Doesn't my feature() commit above make it a breaking change? Do I need to do another one?

Reply:
feature will give it a minor version bump, but we need to do a major version bump to indicate that configs that worked with a previous version won't work with this one -- npm will then require an explicit upgrade

Reply:
Here's an example of a previous one https://github.com/ibi-group/datatools-ui/commit/979f644f5ba0739eda1d3ac9e3180402a6dbf7db